lua/kubectl/actions/buffers.lua

```diff
@@ -163,6 +163,7 @@ end
 --- @param filetype string: The filetype of the buffer.
 --- @param onConfirm function: The function to call on confirmation.
 --- @param opts { syntax: string|nil, content: table|nil, marks: table|nil, width: number|nil }|nil: Options for the buffer.
+--- @return integer, table: The buffer and window number.
```

Comment:
A bit pedantic but it's the window config, not number 😅

Reply:
absolutely right.